sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/FormTrainingAsyncClient.java

```diff
@@ -643,17 +734,22 @@ String getEndpoint() {
 
     private Function<PollingContext<FormRecognizerOperationResult>, Mono<FormRecognizerOperationResult>>
         getTrainingActivationOperation(
-        String trainingFilesUrl, boolean includeSubfolders, String filePrefix, boolean useTrainingLabels,
-        Context context) {
+        String trainingFilesUrl, boolean useTrainingLabels, boolean includeSubfolders, String filePrefix,
+        String modelDisplayName, Context context) {
```

Comment:
should modelDisplayName be modelName? I see it used in some places, so not sure.